// Copyright 2013 <chaishushan{AT}gmail.com>. All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

package hello;

option cc_generic_services = true; // HL

message HelloRequest {
	optional string msg = 1;
}

message HelloResponse {
	optional string msg = 1;
}

service HelloService { // HL
	rpc Hello (HelloRequest) returns (HelloResponse); // HL
}
